src/modules/manager/actions.js

```diff
@@ -39,3 +40,19 @@ export const fetchDropBoxTreeOrFail = () => async dispatch => {
         return await dispatch({type: FETCH_DROP_BOX_TREE.FINISH});
     }
 };
+
+export const putDropBoxObject = networkAction((path, file) => async (dispatch, getState) => ({
+    types: PUT_DROP_BOX_OBJECT,
+    url: `${getState().services.dropBoxService.url}/objects/${path.replace(/^\//, "")}`,
```

Comment:
Is the replace call for url encoding? If so we could use `encodeURI` to prevent more malformed scenarios